hibernate-core/src/main/java/org/hibernate/cache/spi/support/EntityReadWriteAccess.java

```diff
@@ -151,9 +151,4 @@ public boolean afterUpdate(
 	public SoftLock lockRegion() {
 		return null;
 	}
-
-	@Override
-	public void unlockRegion(SoftLock lock) {
```

Comment:
Removing this no-op method here means it will inherit the method from ``AbstractCachedDomainDataAccess``, which calls ``evictAll``.

This is invoked by the ``release`` method in ``BulkOperationCleanupAction.EntityCleanup``, which is called as part of ``doAfterTransactionCompletion``.